src/App/Fossa/EmbeddedBinary.hs

```diff
@@ -109,7 +108,7 @@ writeExecutable path content = do
 extractedPath :: PackagedBinary -> Path Rel File
 extractedPath bin = case bin of
   Syft -> $(mkRelFile "syft")
-  Wiggins -> $(mkRelFile "wiggins")
+  Wiggins -> $(mkRelFile "vsi-plugin")
```

Comment:
If we're changing the name of the executable, I'd like to keep all of the rest of the naming consistent as well.  If this is an issue for timing, as a temporary workaround we can just rename the executable in the downloader script, we currently do this with syft.

Reply:
I changed this because during the VSI strategy we display a line saying something like "[TASK X] Running '/some/temp/path/wiggins'", and my thought was that "wiggins" wouldn't mean anything to users, but "vsi-plugin" would. I left everything else in the code "wiggins" because the name of the binary itself isn't changing, I just wanted it to be displayed differently in that line and didn't see a way to add a more friendly name to the `Command` type.

If you disagree and still think based on that we should rename it globally to `vsi-plugin`, I'm OK to do so & will make that change.

Reply:
Ah I see.  I thought this was also tied to our embedding, but we separate those.  I'm OK with leaving this as you have it, but maybe with a code comment explaining that this is only for users to see more relevant naming.